platforms/org.eclipse.mita.platform.arduinouno/1.0.0/arduinouno.platform

```diff
@@ -0,0 +1,33 @@
+package platforms.arduinouno;
```

Comment:
Sounds good, I'll refactor this.

Reply:
The first contribution is completly realized in C only with AVR libs. Probably it would be interesting how we could integrate the offizial arduino libs, like digitalWrite etc.